index.html

```diff
@@ -150,9 +150,29 @@ <h2 class="content-text">About
             </div>
             
         </div>
+        <footer> 
+            <section class="contact-me">
+                <div class="content">
+                    <h2>
+                        Contact me
+                    </h2>
+                    <p>
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                    <form action="https://formspree.io/f/mknegeno" method="POST">
+                        <input class="name" name="Name" type="text" maxlength="30" placeholder="Becky" required>
+                        <input class="email" type="email" name="email" placeholder="becky@gmail.com" required>
+                        <textarea placeholder="Write Your Message Here" id="sheet" name="message" cols="30" rows="10" maxlength="500" required></textarea>
+                        <button class="btn" value="submit" type="submit">
```

Comment:
- [ ] Kindly add button interactions for active (when it's pressed),  as shown in the Figma template 👍🏿 

![Screenshot_421](https://user-images.githubusercontent.com/69638013/187646915-dc259218-c8eb-43ac-aacd-c9e59be580a8.png)